plugins/cross-links.js

```diff
@@ -1,293 +1,268 @@
 // @author         mcben
 // @name           Cross links
 // @category       Draw
-// @version        1.2.0
+// @version        1.3.0
 // @description    Checks for existing links that cross planned links. Requires draw-tools plugin.
 
 
-window.plugin.crossLinks = function() {};
+window.plugin.crossLinks = function () { };
```

Comment:
What if we make all namespace references shorter?
```js
var crossLinks = window.plugin.window.plugin.crossLinks = function () {};
```

And then:
```js
crossLinks.greatCircleArcIntersect = function...
crossLinks.testPolyLine = function...
// etc
```
BTW, is it necessary to expose all this functions in namespace?
May be better left some of them private?

Reply:
I don't like chained assignments

Reply:
Well, it's actually does not matter.
Consider this:
```js
var crossLinks = function () {};
window.plugin.window.plugin.crossLinks = crossLinks;
```